Workflow failure: cron

## Workflow Failure

[cron](https://github.com/pulumi/pulumi-aws/blob/master/.github/workflows/nightly-test.yml) has failed. See the list of failures below:

- [2025-01-14T08:00:30.000Z](https://github.com/pulumi/pulumi-aws/actions/runs/12762077951)

- [2025-01-14T10:16:18.000Z](https://github.com/pulumi/pulumi-aws/actions/runs/12762077951)

- [2025-01-14T14:48:44.000Z](https://github.com/pulumi/pulumi-aws/actions/runs/12762077951)

`TestRegress2534` failed:

```
=== RUN   TestRegress2534
    provider_test.go:215: installing packages and plugins
    provider_test.go:215: PULUMI_BACKEND_URL=file:///tmp/TestRegress2534536316515/002
    provider_test.go:215: PULUMI_DEBUG_GRPC=/tmp/TestRegress2534536316515/003/grpc.json
    provider_test.go:215: creating stack test
    examples_py_test.go:169: deploying
    examples_py_test.go:173: Provisioned target group with arn=arn:aws:elasticloadbalancing:us-west-2:894850187425:targetgroup/test-8e4404f/81fe1a05e75cabe8
    examples_py_test.go:175: workdir = /tmp/TestRegress2534536316515/001/regress-2534
    provider_test.go:50: stdout: Previewing import (test):
        
        @ previewing import....
         =  aws:lb:TargetGroup newtg import 
        @ previewing import....
            pulumi:pulumi:Stack regress-2534-test  
        Resources:
            = 1 to import
            3 unchanged
        
        Importing (test):
        
        @ importing....
         =  aws:lb:TargetGroup newtg importing (0s) 
        @ importing....
         =  aws:lb:TargetGroup newtg imported (0.78s) 
        Outputs:
            targetGroupArn: "arn:aws:elasticloadbalancing:us-west-2:894850187425:targetgroup/test-8e4404f/81fe1a05e75cabe8"
            targetGroupUrn: "urn:pulumi:test::regress-2534::aws:lb/targetGroup:TargetGroup::test"
        
        Resources:
            = 1 imported
            3 unchanged
        
        Duration: 2s
        
    provider_test.go:51: stderr: error: loading package '&{aws 6.66.3  <nil>}': req: aws@6.66.3: entries: map[aws:6.66.3:0xc0021ad600] (returned aws@6.67.0-alpha.173[681](https://github.com/pulumi/pulumi-aws/actions/runs/12762077951/job/35571523984#step:15:682)4523+bf974ad)
    provider_test.go:52: code=255
    provider_test.go:53: 
        	Error Trace:	/home/runner/work/pulumi-aws/pulumi-aws/examples/provider_test.go:53
        	            				/home/runner/work/pulumi-aws/pulumi-aws/examples/examples_py_test.go:177
        	Error:      	Received unexpected error:
        	            	exit status 255
        	Test:       	TestRegress2534
    provider_test.go:215: cleaning up stack
    provider_test.go:215: failed to destroy stack: failed to destroy stack: exit status 255
        code: 255
        stdout: Destroying (test):
        
        @ destroying....
         -  aws:lb:TargetGroup test deleting (0s) 
         -  aws:lb:TargetGroup test deleted (0.60s) 
         -  aws:ec2:Vpc main deleting (0s) 
         -  aws:lb:TargetGroup newtg deleting (0s) 
         -  aws:lb:TargetGroup newtg deleting (0s) error: resource "urn:pulumi:test::regress-2534::aws:lb/targetGroup:TargetGroup::newtg" cannot be deleted
         -  aws:lb:TargetGroup newtg **deleting failed** error: resource "urn:pulumi:test::regress-2534::aws:lb/targetGroup:TargetGroup::newtg" cannot be deleted
        @ destroying....
         -  aws:ec2:Vpc main deleted (0.84s) 
            pulumi:pulumi:Stack regress-2534-test  error: update failed
            pulumi:pulumi:Stack regress-2534-test **failed** 1 error
        Diagnostics:
          pulumi:pulumi:Stack (regress-2534-test):
            error: update failed
        
          aws:lb:TargetGroup (newtg):
            error: resource "urn:pulumi:test::regress-2534::aws:lb/targetGroup:TargetGroup::newtg" cannot be deleted
            because it is protected. To unprotect the resource, either remove the `protect` flag from the resource in your Pulumi program and run `pulumi up`, or use the command:
            `pulumi state unprotect 'urn:pulumi:test::regress-2534::aws:lb/targetGroup:TargetGroup::newtg'`
        
        Resources:
            - 2 deleted
        
        Duration: 3s
        
        
        stderr: 
        
--- FAIL: TestRegress2534 (36.51s)
```

Retrying...

--- a/examples/examples_py_test.go
+++ b/examples/examples_py_test.go
@@ -165,6 +165,7 @@ func TestRegress3887(t *testing.T) {
 
 // Make sure that importing an AWS targetGroup succeeds.
 func TestRegress2534(t *testing.T) {
+	t.Skipf("TODO[pulumi/pulumi-aws#5106]")
 	ptest := pulumiTest(t, filepath.Join("test-programs", "regress-2534"))
 	upResult := ptest.Up(t)
 	targetGroupArn := upResult.Outputs["targetGroupArn"].Value.(string)

